vllm/v1/core/sched/scheduler.py

```diff
@@ -62,13 +60,31 @@ def __init__(
             self.scheduler_config.max_num_batched_tokens
         self.max_model_len = self.scheduler_config.max_model_len
 
+        # create connector
+        if self.vllm_config.kv_transfer_config is not None:
+            from vllm.distributed.kv_transfer.kv_connector.factory import (
+                KVConnectorFactory)
+            from vllm.distributed.kv_transfer.kv_connector.v1 import (
+                KVConnectorRole as KVConnectorRole_V1)
```

Comment:
nit: Can we somehow avoid the lazy import? Given that we have a factory class, I think this should be doable.